plone/app/tiles/__init__.py

```diff
@@ -1,5 +1,19 @@
 # -*- coding: utf-8 -*-
-import zope.i18nmessageid
+from zope.i18nmessageid import MessageFactory
+from Products.CMFCore.permissions import ManagePortal
 
+_ = MessageFactory('plone')
 
-MessageFactory = zope.i18nmessageid.MessageFactory('plone')
+
+def initialize(context):
+    """Registers modifiers with zope (on zope startup).
+    """
+    from modifiers import modifiers
```

Comment:
I'm always confused about this relative imports, can we turn that into a full package import? It will make later refactorings easier.